docs/getting-setup.md

```diff
@@ -98,38 +98,6 @@ Navigate to `about:config` and accept any warning message. Then search for the f
 
 Close firefox and re-open it with the `firefox-bin` command.
 
-#### Firefox GCLI
-
-1. Open Firefox
```

Comment:
Looks like these changes are from previous PR. Let's remove this `Removed Firefox GLCI information` commit from this PR.